Add XOX70 firmware (#21023)
qmk/qmk_firmware: keyboards/wolf/xox70/hotswap/readme.md

# XOX70_Hotswap

> ![X0X70](https://i.imgur.com/OSW2c6L.png)

Reviewer: ```suggestion
![X0X70](https://i.imgur.com/OSW2c6Lh.png)
```

The following is the QMK Firmware for the XOX70_Hotswap PCB.

* Keyboard Maintainer: [Wolf](https://github.com/ToastyStoemp)
* Hardware Supported: XOX70 Hotswap PCB
* Hardware Availability: https://thocc.supply/products/xox70

Make example for this keyboard (after setting up your build environment):

> 	make wolf/xox70/hotswap:default

Reviewer 2: ```suggestion
    make wolf/xox70/hotswap:default
```

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

> ## Bootloader Enter the bootloader in 1 ways: 
> * **Bootmagic reset**: Hold down the key ESC key and plug in the keyboard (Top Left most switch)

Reviewer: ```suggestion
## Bootloader Enter the bootloader in 2 ways: 
* **Bootmagic reset**: Hold down the key ESC key and plug in the keyboard (Top Left most switch)
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

# XOX70_Hotswap

> ![X0X70](https://cdn.shopify.com/s/files/1/0505/0328/6952/products/WY.jpg?v=1661602119&width=640)

Reviewer 3: image should be rehosted on an image host. Not shopify.
